src/backend/TrafficCourts/Citizen.Service/Validators/LegalRepresentationValidator.cs

```diff
@@ -0,0 +1,22 @@
+﻿using FluentValidation;
+using System.Text.RegularExpressions;
+using TrafficCourts.Citizen.Service.Models.Dispute;
+
+namespace TrafficCourts.Citizen.Service.Validators
+{
+    /// <summary>
+    /// Fluent Validator for Legal Representation Model
+    /// </summary>
+    public class LegalRepresentationValidator : AbstractValidator<LegalRepresentation>
+    {
+        public LegalRepresentationValidator()
+        {
+            RuleFor(_ => _.LawFirmName).NotEmpty();
+            RuleFor(_ => _.LawyerFullName).NotEmpty();
+            RuleFor(_ => _.LawyerEmail).NotEmpty().EmailAddress();
+            RuleFor(_ => _.LawyerPhoneNumber).NotEmpty()
+                .Matches(new Regex(@"\(?\d{3}\)?-? *\d{3}-? *-?\d{4}"));
```

Comment:
I assume only North American 10-digit numbers for now?  Guess this is a business question if ever there would international numbers.

Reply:
Yeah, this only validates North American 10-digit numbers for now. We need to follow up with business to check if we ever need country based validation.